internal/query/projection/projection.go

```diff
@@ -210,6 +213,22 @@ func ProjectInstance(ctx context.Context) error {
 	return nil
 }
 
+func ProjectInstanceFields(ctx context.Context) error {
+	for _, projection := range []*handler.FieldHandler{
```

Comment:
Should this list not better be defined the same as the projections to have consistent?
With a function comparable to newProjectionsList?